versioned_docs/version-2.x/appendices/migration-guides/index.mdx

```diff
@@ -130,6 +130,26 @@ following changes:
 We strongly advice you to synchronize any override you may have done in your
 project.
 
+### Image caching
+
+In this version we've introduced
+[Focal Point](https://gitlab.blackswift.cloud/front-commerce/front-commerce/-/commit/40185d3aecd3fd7c7d959483db16cdea21541a82)
```

Comment:
**issue:** I'm not fan of linking to a commit for this kind of things

**suggestion:** either link to the documentation section (added :point_up:) of Focal point (**my preference**) or to the MR.